modules/33acrossBidAdapter.md

```diff
@@ -10,23 +10,145 @@ Maintainer: headerbidding@33across.com
 
 Connects to 33Across's exchange for bids.
 
-33Across bid adapter supports only Banner at present and follows MRA
+33Across bid adapter supports Banner and Video at present and follows MRA
 
 # Sample Ad Unit: For Publishers
+## Sample Banner only Ad Unit
 ```
 var adUnits = [
 {
-    code: '33across-hb-ad-123456-1', // ad slot HTML element ID   
-    sizes: [
-        [300, 250], 
-        [728, 90]
-    ],     
-    bids: [{
-        bidder: '33across',
-        params: {
-            siteId: 'cxBE0qjUir6iopaKkGJozW',     
-            productId: 'siab'     
+  code: '33across-hb-ad-123456-1', // ad slot HTML element ID  
+  mediaTypes: {
+    banner: {  
+      sizes: [
+          [300, 250], 
+          [728, 90]
+      ]
+    }   
+  } 
+  bids: [{
+    bidder: '33across',
+    params: {
+        siteId: 'sampleGUID123',     
+        productId: 'siab|inview'     
+    }
+  }]
+}
+```
+
+## Sample Video only Ad Unit: Outstream
+```
+var adUnits = [
+{
+  code: '33across-hb-ad-123456-1', // ad slot HTML element ID  
+  mediaTypes: {
+    video: {  
+      playerSize: [300, 250],
+      context: 'outstream',
+      placement: '2'
+      ... // Aditional ORTB video params
+    }   
+  },
+  renderer: {
+    url: 'https://acdn.adnxs.com/video/outstream/ANOutstreamVideo.js',
+    render: function (bid) {
+        adResponse = {
+            ad: {
+                video: {
+                    content: bid.vastXml,
+                    player_height: bid.playerHeight,
+                    player_width: bid.playerWidth
+                }
+            }
         }
-    }]
+        // push to render queue because ANOutstreamVideo may not be loaded yet.
+        bid.renderer.push(() => {
+            ANOutstreamVideo.renderAd({
+                targetId: bid.adUnitCode, // target div id to render video.
+                adResponse: adResponse
+            });
+        });
+    }
+  }, 
+  bids: [{
+    bidder: '33across',
+    params: {
+        siteId: 'sampleGUID123',     
+        productId: 'siab'     
+    }
+  }]
+}
+```
+
+## Sample Multi-Format Ad Unit: Outstream
+```
+var adUnits = [
+{
+  code: '33across-hb-ad-123456-1', // ad slot HTML element ID  
+  mediaTypes: {
+    banner: {  
+      sizes: [
+          [300, 250], 
+          [728, 90]
+      ]
+    },
+    video: {  
+      playerSize: [300, 250],
+      context: 'outstream',
+      placement: '2'
+      ... // Aditional ORTB video params
+    }   
+  },
+  renderer: {
+    url: 'https://acdn.adnxs.com/video/outstream/ANOutstreamVideo.js',
+    render: function (bid) {
+        adResponse = {
+            ad: {
+                video: {
+                    content: bid.vastXml,
+                    player_height: bid.playerHeight,
```

Comment:
not sure the bid.playerHeight|Width is standard, that may exist in APNX adapter only, please double check. if it does not exists, then just suggest with `w & h`